### ISSUE TITLE
NTR: Healthcare activities on admission

### ISSUE THREAD

**Issue by SheenaTomko** [VEuPathDB-ontology/VEuPathDB-ontology#438]
For new term requests, please provide the following information:
**Term Label** [required]
Healthcare activities on admission

**Synonyms** [optional]

**Definition** [required except for organism name]
Multifilter to gather variables about activities that are on the birth checklist and supposed to be performed around admission time

**Suggested Parent Term** [required]
Please look in the ontology hierarchy in a browser such as [OLS](http://www.ebi.ac.uk/ols/ontologies/eupath) or choose one from following list: 
- [ ] Material (independent of other entities and can be maintained through time, such as person, cell, sample)
- [ x] Process (has temporal proper parts and for some time depends on some material entity, such as life cycle, cell division, heart beating, sleeping)
- [ ] Information (generically dependent on some artifact and stands in relation of aboutness to some entity, such as data, identifier, document)
- [ ] Characteristic/Quality (inheres in or is borne by other entities, such as height, color, disease)

**Examples** [optional]

**Contributor** [optional]
If you would like to be a contributor of the term or anyone you think that contributes to the term, such as label, definition, please provide the name and ORCID ID.

**Requester** [optional]
Sheena/ClinEpiDB

**Comment by johnwjudkins** [comment]
Definition:
An information content entity that is about health care encounters that follow admission for baby delivery.

**Comment by SheenaTomko** [comment]
Just want to clarify that this is for encounters/activities that happen during the admission process. The word "follow" in the definition threw me off a bit

**Comment by zhengj2007** [comment]
The subclasses of the term in the clinEpi are:

- Birth checklist picked up after care
- Birth checklist picked up during care
- Birth checklist poster observed during care
- Birth companion present
- Blood pressure measured
- Partograph started
- Temperature measured

The term seems an information rather than plan specification. Besides, we have 'healthcare information' and 'clinical data item'. Is the term a subClass of it? If not, what difference between 'healthcare activity' and OGMS:'healthcare process'?

**Comment by johnwjudkins** [comment]
@SheenaTomko @zhengj2007 Now I think these are just [health care processes](http://purl.obolibrary.org/obo/OGMS_0000096) rather than the more specific [health care encounter](http://purl.obolibrary.org/obo/OGMS_0000097) -- not all these activities have both a health care provider and patient role.

EUPATH:0061767
label: information on health care during admission for baby delivery
definition: An information content entity that is about other health care processes during admission for baby delivery.

**Comment by SheenaTomko** [comment]
I would agree that this is about a health care process rather than encounter. I like the update to the definition

**Comment by zhengj2007** [comment]
Is it subClassOf [healthcare information](http://purl.obolibrary.org/obo/EUPATH_0000451)?

What does 'other health care processes' mean?
